python/skupper_router/management/skrouter.json

```diff
@@ -650,6 +650,12 @@
                     "required": false,
                     "create": true
                 },
+                "version": {
+                    "description":"Build version of skupper",
```

Comment:
Maybe make this a little more descriptive:  "Build version of the running control-plane software"

Reply:
> Maybe make this a little more descriptive: "Build version of the running control-plane software"

Done !